installer.py

```diff
@@ -71,10 +72,10 @@ def upgradeFiles():
     os.system(subbruteUpgrade)
     info("\nSubbrute Installed\n")
 
-    amassUpgrade = "go get -v -u github.com/OWASP/Amass/v3/..."
+    amassUpgrade = "GO111MODULE=on go get -v -u github.com/OWASP/Amass/v3/..."
     info("\nInstalling Amass ")
     os.system(amassUpgrade)
-    subfinderUpgrade = "go get -u -v github.com/projectdiscovery/subfinder/cmd/subfinder"
+    subfinderUpgrade = "GO111MODULE=on go get -u -v github.com/projectdiscovery/subfinder/cmd/subfinder"
```

Comment:
Why do we need `GO111MODULE=on`?

Reply:
Setting that variable enables Go Modules, that allows to select versions. In particular, attempting to install Amass in latest Parrot without it, results in:
```
go get -v -u github.com/OWASP/Amass/v3/...
github.com/OWASP/Amass (download)
go: warning: "github.com/OWASP/Amass/v3/..." matched no packages
```

When setting that environment variable:
```
GO111MODULE=on go get -v -u github.com/OWASP/Amass/v3/...
go: finding github.com/gobwas/httphead latest
go: finding golang.org/x/sys latest
go: finding golang.org/x/net latest
go: finding github.com/geziyor/geziyor latest
go: finding golang.org/x/sync latest
go: finding golang.org/x/tools latest
...
```

I guess this could be useful for other installations as well, though I'm no expert in Go, so feel free to discuss/remove this.

Ref: https://dev.to/maelvls/why-is-go111module-everywhere-and-everything-about-go-modules-24k